src/kontrol/__main__.py

```diff
@@ -419,7 +420,9 @@ def parse(s: str) -> list[T]:
 
     command_parser = parser.add_subparsers(dest='command', required=True)
 
-    command_parser.add_parser('version', help='Print out version of Kontrol command.')
+    command_parser.add_parser(
+        'version', help='Print out version of Kontrol command.', parents=[kontrol_cli_args.logging_args]
```

Comment:
Do we need `logging_args` here? Running `kontrol version` with `--debug` or `--verbose` seems to produce the same output as without args, but maybe it's good to keep it consistent with other subcommands.

Reply:
There is a problem is with the logic to decide the logging level. Anyway, I modified that part of the code to give the default logging level is neither 'debug' or 'verbose' is specified.